en/frames_multicopter/holybro_x500V2_pixhawk5x.md

```diff
@@ -0,0 +1,223 @@
+- **Assembly time (approx.):** 55 min (25 minutes for frame, 30 minutes for autopilot installation/configuration)
+
+![Full X500 V2 Kit](../../assets/airframes/multicopter/x500_v2_holybro_pixhawk5x/x500-kit.png)
+
+## Bill of materials
+
+The Holybro [X500 V2 Kit](http://shop.holybro.com/x500-v2-kit_p1288.html) includes almost all the required components:
+
+* X500V2 Frame Kit
+  * Body - Full Carbon Fiber Top & Bottom plate (144 x 144mm, 2mm thick)
+  * Arm - High strength & ultra-lightweight 16mm carbon fiber tubes
+  * Landing gear - 16mm & 10mm diameter carbon fiber tubes
+  * Platform board - With mounting holes for GPS & popular companion computer
+  * Dual 10mm Ø rod x 250 mm long rail mounting system
+  * Battery mount with two Battery Straps
+  * Hand tools for installation
+* [Holybro Motors - 2216 KV880 x6](https://shop.holybro.com/motor2216-880kv-1pc_p1154.html)
+* [Holybro BLHeli S ESC 20A x4](https://shop.holybro.com/blheli-s-esc-20a_p1143.html)
+* [Propellers - 1045 x4](https://shop.holybro.com/propeller10452pair_p1155.html)
+* Power Distribution Board – XT60 plug for battery & XT30 plug for ESCs & peripherals
+* Camera mount (optional and the 3D file can be downloaded from [here](http://www.holybro.com/3D_Print/Holybro_X500_V2_3D%20Print.rar))
+
+Other parts in this build:
+* [Pixhawk 5X autopilot](../flight_controller/pixhawk5x.md)
+* [M8N GPS](http://shop.holybro.com/holybro-m8n-gps_p1094.html)
+* [Power Module - PM02D](http://shop.holybro.com/pm02d-power-module_p1285.html)
+* [433/915 MHz Telemetry Radio](http://shop.holybro.com/sik-telemetry-radio-v3_p1103.html)
+
+Additionally you will need a battery and receiver ([compatible radio system](../getting_started/rc_transmitter_receiver.md)) if you want to control the drone manually.
+
+## Main Hardware
+
+This section lists all hardware for the frame and the autopilot installation.
+
+Item | Description | Quantity
+---|---|---
+Bottom plate                  |  Carbon fiber (2mm thick)                             | 1
+Top plate                     |  Carbon fiber (1.5mm thick)                           | 1
+Arm                           |  Carbon fiber tube (Assembled with motors mounted)    | 4
+Landing gear - Vertical pole  |  Carbon fiber tube + engineering plastic              | 2
+Landing gear - Cross bar      |  Carbon fiber tube + engineering plastic + foam       | 2
+Mounting Rail                 |  Diameter: 10mm length: 250mm                         | 2
+Battery mounting board        |  Thickness: 2mm                                       | 1
+Battery pad                   |  3mm Silicone sheet black                             | 1
+Platform board                |  Thickness: 2mm                                       | 1
+Hanger & rubber ring gasket   |  Inner hole diameter: 10mm black	                    | 8
+
+### ![X500V2 ARF Kit Full Package Contents](../../assets/airframes/multicopter/x500_v2_holybro_pixhawk5x/x500_v2_whats_inside.png)
```

Comment:
Why is this image in a heading?

Reply:
Really why??? :smile:  Removed from the heading.